Remove 'API Endpoints' section from 'README' & move to 'pages' directory.

ThirDecade2020 opened the pull request:
Removed 'API Endpoints' section and placed in new file called 'apiEndpoints.md' in 'pages' directory.

ThirDecade2020 committed:
Create apiEndpoints.md in 'pages' directory.

Create apiEndpoints.md in 'pages' directory to clean up README .

ThirDecade2020 committed:
Remove 'API Endpoints' section.

Remove 'API Endpoints' section and move to new file in 'pages' directory called 'apiEndpoints.md'.

ykdojo commented:
Can you add a link to the new page from the main README?

ThirDecade2020 committed:
Add link to 'pages/apiEndpoints.md' from 'README.md'

This link allows users to go from main 'README.md' to 'pages/apiEndpoints.md' if they want to see endpoint code to be used for API calls.

ykdojo commented on README.md:
```diff
@@ -59,6 +41,10 @@ The project provides several API endpoints that allow you to interact with the f
 
+## API Endpoints
+
+View API endpoints that allow you to interact with the file system within the Kaguya directory [here](pages/apiEndpoints.md).  
```
We should specify that these interact with the FILES directory by default

ThirDecade2020 replied:
OK will add.

ThirDecade2020 committed:
Specify that API Endpoints Interact with FILES Directory

Linked API Endpoints & FILES Directory to their respective locations

Noted that it is default

ykdojo commented on README.md:
```diff
@@ -43,7 +43,7 @@ When you want to use executeCommand in a subdirectory, make sure to cd there fir
 ## API Endpoints
 
-View API endpoints that allow you to interact with the file system within the Kaguya directory [here](pages/apiEndpoints.md).  
+View [API endpoints](https://github.com/ThirDecade2020/kaguya/blob/master/pages/apiEndpoints.md) that allow you to interact with the [FILES directory](https://github.com/ThirDecade2020/kaguya/tree/master/FILES) by default.
```
This link should be formatted so it works when it's merged

ThirDecade2020 replied:
OK had a thought and I see what's happening here some q's if you don't mind?

1) All my links throughout the pull request are wrong because they are either incomplete or link user to my forked version of the repository.

2) 1 link should take user to a newly created 'apiEndpoints.md', and 1 link should take user to 'FILES' which already existed pre-PR. 'FILES' link is easy to find in original Kaguya, but newly created 'apiEndpoints.md' won't appear in original Kaguya until after merge. My question is, should we insert a link into a PR that goes to a page that won't be created until after the merge? Should I do separate pull requests, A. create new page first, and then B. create link to that new page?

3) Lastly, I didn't realize this, but at the beginning of this PR I did not add a new branch to my version of the repository so I am working directly from my Main branch. Should I start this PR over and create a new branch for each individual change? 

Your help and patience is much appreciated and I'll continue working on this.

ThirDecade2020 commented:
Sorry I cut out for a while life and work increased so time decreased but I knew I would be back at some point to check-in/collaborate again. How are things going with Kaguya?

ykdojo commented:
I haven't had too much time to maintain it recently.

I actually tried setting up a simple ChatGPT plugin that should work locally but it didn't... not sure if OpenAI changed things on their end
